buildscripts/hpat-conda-recipe/meta.yaml

```diff
@@ -21,24 +21,21 @@ requirements:
     - pandas
     - mpich # [not win]
     - pyarrow 0.9.*
-    - boost
+    - boost-cpp
     - hdf5
     - h5py
-    - daal-devel
     - opencv 3.3.1
-    - hpat-parquet-reader # [not win]
```

Comment:
We need hpat-parquet-reader for building with default Anaconda.

Reply:
Why?